xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeMoviesOverview.cpp

```diff
@@ -24,7 +24,8 @@ Node MovieChildren[] = {
                         { NODE_TYPE_STUDIO,       "studios",    20388 },
                         { NODE_TYPE_SETS,         "sets",       20434 },
                         { NODE_TYPE_COUNTRY,      "countries",  20451 },
-                        { NODE_TYPE_TAGS,         "tags",       20459 }
+                        { NODE_TYPE_TAGS,         "tags",       20459 },
+                        { NODE_TYPE_TYPES,        "types",      39300 },
```

Comment:
Some places it is "version" others "types", could help to use just one term throughout. Both version and type could be a variety of other things, how about "release" instead of either?

Reply:
I wanted to use version everywhere, but we already have a version table in db, so I chose (version) types

Reply:
Yes I understand, and sorry it is so late in the day to have this discussion on terminology.  But "version" is already in use, "type" is very generic and mixing terms for one feature even more confusing, hence  suggesting finding another unique term that can be used everywhere.

Reply:
I think "release" is good to use in code/db/api, but for UI, I still think "version" is better. What do you think?

Reply:
Or I just rename the version table (there is only a db version number there), and use "version" everywhere for movie version?

Reply:
Renaming of version table is not viable. The version table is used by the db migration processing, common to all Kodi db.

Reply:
"Version" in the UI doesn't sound quite right to me, but I can't describe why.

Tossing another possibility into the hat, how about "[edition](https://www.merriam-webster.com/dictionary/edition)"?

Reply:
@rmrector Thanks for sharing this with me. English is not my native language, I don't know which one is better, "Version" or "Edition". Seems people use "Version" sometimes (https://en.wikipedia.org/wiki/Versions_of_Blade_Runner), and use "Edition" other times (https://en.wikipedia.org/wiki/Star_Wars_(film)). I like "Edition" personally, but seems "Version" is used more widely, this is why I chose "Version" for this PR. I'm OK with any of these though. Glad to get more opinions from native more speakers.